README.md

```diff
@@ -87,6 +87,26 @@ and `nvidia-docker run -v  $(pwd):/repo closerlookfewshot [command]`,
 e.g. `nvidia-docker run -v  $(pwd):/repo closerlookfewshot python3 /repo/train.py --dataset CUB --model Conv4 --method baseline --train_aug`.
 Change the CUDA version in `10.2-cudnn7-runtime-ubuntu16.04` (`Dockerfile-gpu`) if you have another version than 10.2.
 
+## Protonetpp 
+Deal with domain shift, backbone learns novel features.
+
+### Prepare datasets
+3-way train, 2-way test
+base_classes = ["beaver", "dolphin", "otter"]; 
+val_classes = ["beaver", "dolphin", "otter"];
```

Comment:
val classes should be different from base classes. Otherwise we validate on the same data as we train on.